### DISCUSSION TITLE
Add right-click "copy value clipboard" option for all field displays

### DISCUSSION THREAD

**Discussion by alexbrowng** [directus/directus#8535] [Feature Requests]
In the table view, if the field type allows it, add a button that allows you to copy the text of the field without having to enter the view, it could be a button that appears at the end or beginning of the cell when you hover over the field or that is always visible on touch screens, perhaps, if it is too heavy at the interface level, you can achieve the same functionality with "Double click" or "Double tap" or CTRL + Click.

**Comment by benhaynes** [comment]
I think this is a good _global_ option, since that way it can be used with any existing displays. Maybe we use the right-click (and touch equivalent) to access this?

Similarly, we should add `Copy Value` to the field label dropdown... I'll open a new discussion for that.

**Reply by rijkvanzanten** [reply]
> Maybe we use the right-click (and touch equivalent) to access this?

I don't think there's a good touch equivalent nowadays. Tap-and-hold on iOS triggers "quick actions"

**Reply by benhaynes** [reply]
Does iOS support differentiating double-tap?

**Reply by rijkvanzanten** [reply]
Double tap zooms in.. Don't think you can hard-disable that anno 2021, but we'll have to research that some more

**Reply by alexbrowng** [reply]
It is just an idea, but, if you change the way you enter the detail view of an item, for example, in Notion, there is the inline edition and if you want to see the detail view a specific icon appears for that and depending on the type of field you have inline options for that field.

**Reply by rijkvanzanten** [reply]
Yeah good one. It's a difficult question, cause the same table row could now effectively be used for 3 different actions (navigate to detail, select row, copy value).

**Reply by alexbrowng** [reply]
You could improve the individual actions, to delete for example, now you have to select the row and look for the up button, but, with another model you could still delete a row "faster", maybe a context menu with options, like for example duplicate...

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
